src/hyperfine/export/markdown.rs

```diff
@@ -0,0 +1,38 @@
+use super::{ExportEntry, Exporter};
+
+use std::io::Result;
+
+const MULTIPLIER: f64 = 1e3;
```

Comment:
In the `format.rs` modules, we have some functions that could help here. We could first determine if we want to print all results in seconds or milliseconds and then call `format_duration(..., Unit::Second)` or `format_duration(..., Unit::MilliSecond)`. I could also look into this separately, if you prefer.

Reply:
I originally used them, but they format to strings with "ms" on them, since we put ms in the header, the table looks cleaner with only numbers in the cells, and a change to those functions would impact more of the core code, I just brought over the math.

Reply:
You are right. We would probably have to restructure `format` a bit and introduce another variant that just returns the number. I can take a look at this at a later stage.